microsoft/testsuites/core/vm_hot_resize.py

```diff
@@ -37,14 +39,30 @@ class VmHotResize(TestSuite):
             supported_features=[Resize],
         ),
     )
-    def verify_vm_hot_resize(self, node: Node) -> None:
+    def verify_vm_hot_resize(self, log: Logger, node: Node) -> None:
         resize = node.features[Resize]
-        expected_vm_capability = resize.resize()
+        retry = 1
+        maxretry = 10
+        while retry < maxretry:
+            try:
+                expected_vm_capability = None
+                expected_vm_capability = resize.resize()
+                break
+            except Exception as e:
+                if "cannot find current vm size in eligible list" in str(e):
+                    retry = retry + 1
+                elif "OperationNotAllowed" in str(e):
+                    retry = retry + 1
+                else:
+                    log.debug("unexpected exception found")
```

Comment:
For unknown exception, use `raise identifier` to reraise it.

Reply:
update